annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -108,6 +110,7 @@ protected void validateGeoTag(Body body) throws ParamValidationException {
     }
 
     protected void validateTagWithSpecificResource(Body body) throws ParamValidationException {
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.